.github/actions/build_ios/action.yml

```diff
@@ -0,0 +1,64 @@
+name: Build iOS app
+description: Build and upload iOS app
+inputs:
```

Comment:
We should verify that secret values are not exposed when they are parsed as inputs

Reply:
Github Actions redacts secrets automatically regardless of how they are passed around. From the logs:

2023-10-29T10:31:45.9020320Z ##[group]Run .github/scripts/setup_certs.sh
2023-10-29T10:31:45.9021020Z [36;1m.github/scripts/setup_certs.sh[0m
2023-10-29T10:31:45.9156510Z shell: /bin/bash --noprofile --norc -e -o pipefail {0}
2023-10-29T10:31:45.9157150Z env:
2023-10-29T10:31:45.9157490Z   PROD_RELEASE: false Environment variable
2023-10-29T10:31:45.9157890Z   TAG_NAME: dev <-- Environment variable
2023-10-29T10:31:45.9204640Z   APPLE_IOS_SIGNING_CERT: *** <-- Secret

2023-10-29T10:31:45.9205180Z   APPLE_IOS_SIGNING_CERT_PW: *** <-- Secret
2023-10-29T10:31:46.8769810Z   APPLE_IOS_PROVISIONING_PROFILE: *** <-- Secret

2023-10-29T10:31:46.8770530Z   APPLE_KEYCHAIN_PW: *** <-- Secret

Reply:
<3